picky/CHANGELOG.md

```diff
@@ -49,7 +49,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `PrivateKey::from_ec_der_with_curve`
 - `PrivateKey::from_ec_der_with_curve_oid`
 - `PrivateKey::generate_ec`
-- `PrivateKey::from_encoded_ec_components`
```

Comment:
Renamed to be more consistent with other similar methods (`encoded` word now goes _after_ algo name)

Reply:
Sounds good